### ISSUE TITLE
Server Mob spawn rate issue

### ISSUE THREAD

**Issue by Wiral2** [Lycanite/LycanitesMobs#403]
I recently updated to the most updated version and I still have way too many lycanite mobs spawning on my server. With 3 people there's about 250-300 mobs on the server and it causes some major frame drops because of this. Is there a way that I can fix this? I tried making the weight scale "0.25" and mob type limit "10" but it didn't help at all. The primary problem is that if players are grouped together it spawns way too many mobs at that location. For example if there are 3 players in the area it'll be around 300 mobs in that area. This is on version 1.20.0.1. I also get this spammed on my console quite often and I'm not sure what it is:  https://imgur.com/a/aWy6kWa

**Comment by Lycanite** [comment]
The errors in the console are caused by Sponge and Eyewigs, so Sponge is checking Eyewigs if they are hostile before spawning but Eyewigs haven't been given a world yet meaning they can't check the light level, it's just a warning that gets ignored though but will be fixed next update.

Have you made changes to the mob spawners such as sky.json?

**Comment by Wiral2** [comment]
Yeah I made changes to to the spawners. I think I disabled the Ooze spawner and  made lava and fire spawner only work in the nether

**Comment by Lycanite** [comment]
Ah, can you make sure that all passive spawners (fire, lava, ooze, etc but not ore, tree, crop, etc) are changed from using "tick" to using "world" as a trigger type.
You might also be better off clearing them and starting with fresh ones.
"tick" which can also be entered as "player" works on player ticks.
"world" is a new trigger introduced where it works on world ticks and searches for groups of players within 100 blocks of each other.

**Comment by Wiral2** [comment]
So here are things I've done to test:
I tried reseting my config file for the server and seeing if that changes anything,

**Comment by Wiral2** [comment]
So I did reset my config file, it didn't really change too much. But it actually might have been Sponge server thats causing mob spawn issues on the server. I have to test with more of my friends when they get on.

**Comment by Lycanite** [comment]
Ok, let me know how it goes, if you still have overspawning, could you send me a list of some of the mobs that are too frequent too.
